### DISCUSSION TITLE
Explain monetization / company direction in the readme

### DISCUSSION THREAD

**Discussion by Gaibhne** [nocodb/nocodb#6871] [General]
### Please confirm if feature request does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

I would like to use this software, but it is unclear to me what I am getting into. It seems the software is free ... maybe ? On the homepage I found no information about that at all, and nothing clear in the GitHub project readme either. It appears that there is a commercial entity behind this project, but it is totally unclear how it is monetized/profitable.

With the lack of information available, I am unable to ascertain whether adopting this software will lead us down a path where we are charged something, how much, or whether money is made by selling any of my information or other typical monetization schemes.

### Suggested Solution

Add a section to the readme (and the website) about:

* whether the software is free and is intended to stay that way, or alternatively a price/feature matrix
* how the company behind the software is intending to make money off of us using it

### Additional Context

Being open about what drives the development of NocoDB seems like a good way to build trust in users.

**Comment by fulldecent** [comment]
Please see this other self hosted / open source project that has a path to revenue using modules:

Project homepage: https://freescout.net/

Modules: https://freescout.net/modules/

# Here's how it works:

Installing the main product is great

The main product is slim and does not have a million nice features you want

To get those features, visit the website and buy an activation code (one-time purchase)

Customers turn on those modules with the admin page and enter the activation code

The Free Scout project uses PHP/Laravel so the implementation details of how they code that are not helpful here. But the design is usable for our project here. Modules are all first-party and directly benefit the maintainers of the project. Later they may add a module marketplace.

NocoDB can consider this model for sustainably funding the project.

The first module can be to remove the "sponsor us" button, GitHub/Demo links and other things that are not needed in a production environment.

**Comment by ovizii** [comment]
Since I am also interested in this issue, I did some research. Here is what I can contribute to the initial question:

[Here is](https://docs.nocodb.com/FAQs#what-is-available-in-free-version-) what their FAQ says about the free version: 

![image](https://github.com/nocodb/nocodb/assets/837902/e82c78f0-c5e2-448e-ae46-67dd90ec37ce)


With a much more [detailed link](https://github.com/orgs/nocodb/projects/13/views/2) which seems to hint that indeed all current features are free except they also levy the [SSO tax](https://sso.tax/) although they are not lsited yet.
